database/queries.ts

```diff
@@ -78,12 +80,10 @@ export const checkStreak = async () => {
   }
 
   const currentTime = Math.floor(Date.now() / 1000);
-
-
   const result = calculateStreak(entryTimes, currentTime, STREAK_WAIT_TIME_IN_SECONDS, STREAK_BREAK_TIME_IN_SECONDS);
-  console.log(`Current streak: ${result}`);
-
-  setTimeout(checkStreak, AUTOCHECK_INTERVAL_IN_SECONDS * 60000); 
+  if (loopRecursively === true){
+    setTimeout(checkStreak, AUTOCHECK_INTERVAL_IN_SECONDS * 60000); 
+  }
```

Comment:
This seems to just delay a single update's return, rather than return values periodically.

I'm wondering why we need to continually check streak. To me, I feel like we can just check it whenever it's displayed